docs/install_qsimcirq.md

```diff
@@ -1,49 +1,52 @@
 # Installing qsimcirq
 
-The qsim-Cirq Python interface is available as a PyPI package for Linux users.
-For all other users, Dockerfiles are provided to install qsim in a contained
+The qsim-Cirq Python interface is available as a PyPI package for Linux, MacOS and Windows users.
+For all others, Dockerfiles are provided to install qsim in a contained
 environment.
 
 **Note:** The core qsim library (under
 [lib/](https://github.com/quantumlib/qsim/blob/master/lib)) can be included
 directly in C++ code without installing this interface.
 
-## Linux installation
+## Before installation
 
 Prior to installation, consider opening a
 [virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/).
 
-The qsim-Cirq interface uses [CMake](https://cmake.org/) to ensure stable
-compilation of its C++ libraries across a variety of Linux distributions.
-CMake can be installed from their website, or with the command
-`apt-get install cmake`.
-
-Other prerequisites (including pybind11 and pytest) are included in the
+Prerequisites are included in the
 [`requirements.txt`](https://github.com/quantumlib/qsim/blob/master/requirements.txt)
 file, and will be automatically installed along with qsimcirq.
 
-To install the qsim-Cirq interface on Linux, simply run `pip3 install qsimcirq`.
-For examples of how to use this package, see the tests in
-[qsim/qsimcirq_tests/](https://github.com/quantumlib/qsim/blob/master/qsimcirq_tests/).
+## Linux installation
+
+We provide `qsimcirq` Python wheels on 64-bit `x86` architectures with `Python 3.{5,6,7,8,9}`.
```

Comment:
Cirq, which is a requirement for qsimcirq, does not support Python 3.5. Any existing qsimcirq-level support for Python 3.5 (e.g. CI testing) can be dropped.

Reply:
ah yes, that makes sense, updated the PR